WD-Firmware/rtos/rtosal/src/rtosal_util.c

```diff
@@ -30,7 +30,10 @@
 #include "rtosal_macro.h"
 #ifdef D_USE_FREERTOS
```

Comment:
rtosalParamErrorNotifyRunRegister to be move to rtosal_error.c
rtosalEndScheduler and rtosal_start to move to rtosal_task.c

rtosal_utili.c will be of local use only, double check if need to move other stuff.